openbb_terminal/stocks/stocks_models.py

```diff
@@ -225,3 +229,61 @@ def load_stock_polygon(
     df_stock_candidate = df_stock_candidate.sort_values(by="date")
     df_stock_candidate = df_stock_candidate.set_index("date")
     return df_stock_candidate
+
+
+def load_quote(symbol: str) -> pd.DataFrame:
+    """Ticker quote.
+
+    Parameters
+    ----------
+    symbol : str
+        Ticker
+    """
+    ticker = yf.Ticker(symbol)
+
+    try:
+        quote_df = pd.DataFrame(
+            [
+                {
+                    "Symbol": ticker.info["symbol"],
+                    "Name": ticker.info["shortName"],
+                    "Price": ticker.info["regularMarketPrice"],
+                    "Open": ticker.info["regularMarketOpen"],
+                    "High": ticker.info["dayHigh"],
+                    "Low": ticker.info["dayLow"],
+                    "Previous Close": ticker.info["previousClose"],
+                    "Volume": ticker.info["volume"],
+                    "52 Week High": ticker.info["fiftyTwoWeekHigh"],
+                    "52 Week Low": ticker.info["fiftyTwoWeekLow"],
+                }
+            ]
+        )
+
+        quote_df["Change"] = quote_df["Price"] - quote_df["Previous Close"]
+        quote_df["Change %"] = quote_df.apply(
+            lambda x: f'{((x["Change"] / x["Previous Close"]) * 100):.2f}%',
+            axis="columns",
+        )
+        for c in [
+            "Price",
+            "Open",
+            "High",
+            "Low",
+            "Previous Close",
+            "52 Week High",
+            "52 Week Low",
+            "Change",
+        ]:
+            quote_df[c] = quote_df[c].apply(lambda x: f"{x:.2f}")
+        quote_df["Volume"] = quote_df["Volume"].apply(lambda x: f"{x:,}")
+
+        quote_df = quote_df.set_index("Symbol")
+
+        quote_data = transpose(quote_df)
```

Comment:
This would make me happier if you made this quote_df.T and get rid of the import